npm-debug.log

```diff
@@ -0,0 +1,126 @@
+0 info it worked if it ends with ok
```

Comment:
Let's not check this guy in.  So to remove it, just do:
`git rm npm-debug.log`
Then add it to .gitignore